src/Exec_SequenceAlign.cpp

```diff
@@ -153,10 +153,11 @@ Exec::RetType Exec_SequenceAlign::Execute(CpptrajState& State, ArgList& argIn) {
       if (Query[qres] == Sbjct[sres]) { // Exact match. All non-H atoms.
         for (int qat = QR.FirstAtom(); qat != QR.LastAtom(); qat++)
         {
-          if (qref.Parm()[qat].Element() != Atom::HYDROGEN)
```

Comment:
I don't get how this was ever compiling - thanks. I really have to watch these "hidden" commands that have no test cases. They should be completely disabled or stashed in a branch until they are finished.

Reply:
It's legal syntax in C and C++, right?  If a control block doesn't have braces, then the implicit code block is 1 command.  At least that's my understanding.

So this code would call `sTop.AddTopAtom( qref.Parm()[qat], SR );` only if the element was not hydrogen, but would *always* call `sFrame.AddXYZ( qref.Coord().XYZ(qat) );`, even if the atom was hydrogen.

I only saw this because the compiler warning spit out something about deceptive indentation :).  The fact that cpptraj's build is quite clean is A Very Good Thing (TM).